bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -571,3 +571,5 @@ R00076,R00076,Sensor,0,CE09OSPM-00003 RTE000000,CE09OSPM-00003 RTE000000,,,CE09O
 R00077,R00077,Sensor,0,CE09OSPM-00003 RTE000000,CE09OSPM-00003 RTE000000,,,CE09OSPM-00004 STCENG000,,20130122,,,
 R00078,R00078,Sensor,0,CE04OSPD-DP01B-00001-ENG,CE04OSPD-DP01B-00001-ENG,,,CE04OSPD-DP01B-00001-ENG,,20140816,,,
 R00079,R00079,Sensor,0,CE04OSPD-DP01B-00002-ENG,CE04OSPD-DP01B-00002-ENG,,,CE04OSPD-DP01B-00002-ENG,,20150724,,,
+CGCON-NDCL27-00033,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,33,,,,,Woods Hole Oceanographic Institution
+CGCON-BSTC01-00040,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0040,,,,,Woods Hole Oceanographic Institution
```

Comment:
Should these be up with the rest of the `CGCON-*` entries (they would go up to approx lines 338 and 102)? I'm sure the code doesn't care, but for consistency.

Reply:
Rebecca put them at the bottom so that they would be easier to find because she thought there might be an issue with them given old vs. new S/Ns.   I think we should put them where they belong and put a note at the end.  I'll do that.

Reply:
Updates look good. Thanks,